tools/autograd/gen_autograd.py

```diff
@@ -42,21 +42,28 @@
     'as_strided': 'self',
     'diagonal': 'self',
     'expand': 'self',
+    'expand_as': 'self',
     'narrow': 'self',
     'permute': 'self',
+    'reshape': 'self',
+    'reshape_as': 'self',
     'select': 'self',
     'slice': 'self',
     'squeeze': 'self',
     't': 'self',
     'transpose': 'self',
     'unfold': 'self',
     'unsqueeze': 'self',
+    'flatten': 'flatten',
     'view': 'self',
+    'view_as': 'self',
     'unbind': 'self',
     '_indices': 'self',
     '_values': 'self',
     'indices': 'self',
     'values': 'self',
+    'contiguous': 'self',
```

Comment:
Can you move this and the reshape to the list below please.
These are composite ops that should not be in this list.

Reply:
Done.